pkg/jiraloader/jiraloader.go

```diff
@@ -0,0 +1,114 @@
+package jiraloader
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strconv"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	v1jira "github.com/openshift/sippy/pkg/apis/jira/v1"
+	"github.com/openshift/sippy/pkg/db"
+	"github.com/openshift/sippy/pkg/db/models"
+)
+
+type JIRALoader struct {
+	dbc *db.DB
+}
+
+func New(dbc *db.DB) *JIRALoader {
+	return &JIRALoader{
+		dbc: dbc,
+	}
+}
+
+func (jl *JIRALoader) LoadJIRAIncidents() error {
+	start := time.Now()
+	log.Infof("fetching incidents from jira...")
+
+	/* Note a token isn't currently required to hit the issues.redhat.com API. This gets
+	   us public jira cards which is probably what we want, I don't think we're ever doing non-public
+	   incidents. That way we don't leak anything embargoed. If at some point we do need a token, you
+	   can do it with the below, but make sure it has limited privileges and can only see public cards.
+
+		token := os.Getenv("JIRA_TOKEN")
+		req.Header.Add("Authorization", "Bearer "+token)
+	*/
+
+	client := &http.Client{}
+	req, err := http.NewRequest("GET", "https://issues.redhat.com/rest/api/2/search?jql=labels%20in%20(trt-incident,trt-resolved)", nil)
```

Comment:
I know we had discussed trt-resolved but I believe we decided to just clone incidents and keep one for our board, closing when done and the other wouldn't have trt-incident on it.  Just mentioning in case we don't want to include trt-resolved at this time.

Reply:
Fixed